### ISSUE TITLE
Can't receive notification and unreadcountmessage only triggered when the app in conversation page

### ISSUE THREAD

**Issue by rezyfr** [freshworks/freshchat-android#105]
I just set up Freshchat on my app and it works just fine. However the notification doesn't work, I also log the unreadcountmessage but it only triggered when I'm on the conversation page. I injected the Freshchat initiation into my App class. The "Push Notification Enabled" on user properties is also said 'Yes'. Here is how I handle my Freshchat.
![image](https://user-images.githubusercontent.com/22698234/93310008-6c74cf00-f82e-11ea-9642-3cfa92cc8731.png)
![image](https://user-images.githubusercontent.com/22698234/93310092-89a99d80-f82e-11ea-9721-6a878fcb6195.png)

this code is on my MainActivity but it also only logged while I'm on the conversation page
![image](https://user-images.githubusercontent.com/22698234/93310454-fde44100-f82e-11ea-9c16-be95b7522e3f.png)

and I already write the provider for Freshchat and service for FCM on the manifest file.

I already have a ticket on support.freshchat.com but it takes forever to get a reply. Any idea what did I do wrong

here's a snippet of the response
...
"pushSuccessOnServer":false,"pushEnabled":true,"pushFailureReason":"The notification ID generated on the device and the GCM key on the server do not match","pushFailureRemedy":"Ensure that the GCM project ID on the device matches with the GCM server key. Also double check that the GCM server key on Hotline.io is correct.","gcmNotificationResult":{"result":{"errorCode":"MismatchSenderId"}}}

**Comment by bboyzlodey** [comment]
Hello @rezyfr !

**Push notifications**
I init a push notification in Aplication and it work:
```kotlin
  private fun setPushNotificationsFreshchat() {
        FirebaseInstanceId.getInstance().instanceId.addOnCompleteListener {
            if (it.isSuccessful) {
                Freshchat.getInstance(applicationContext).setPushRegistrationToken(it.result.token)
            }
            Timber.tag(javaClass.simpleName).d("Task 'instanceId ${it.isSuccessful}")
            Timber.tag(javaClass.simpleName).d("Task 'instanceId ${it.result.token}")
        }

        val notificationConfig = FreshchatNotificationConfig()
            .setNotificationSoundEnabled(true)
            .setPriority(NotificationCompat.PRIORITY_HIGH)
        Freshchat.getInstance(applicationContext).setNotificationConfig(notificationConfig)
    }
```
**getUnreadCountAsync**
Read [here](https://github.com/freshworks/freshchat-android/issues/89#issuecomment-692906853)

That's all I can do to help

**Comment by MuthuFD** [comment]
@rezyfr Can you confirm if you have followed and implemented section 6.1 from the integration documentation.
https://support.freshchat.com/support/solutions/articles/50000000207-freshchat-android-sdk-integration-steps

Only if push notifications works, unread count will be in sync.

**Comment by MuthuFD** [comment]
@rezyfr Please refer below video tutorials for more details.
Android - Push Notification Integration - https://freshchat.wistia.com/medias/69v54tgg81

Hope this solves your issue. I'm closing this ticket. Feel free to reopen or contact us at support@freshchat.com if you are still this issue.
